### ISSUE TITLE
Font

### ISSUE THREAD

**Issue by ColoringNuns** [UnsignedByte/gunndertale#15]
Here's a font I think might fit a bit better. It needs no credit or anything so you should be good to just substitute it.
[Prstart.ttf.zip](https://github.com/UnsignedByte/gunndertale/files/2895261/Prstart.ttf.zip)

**Comment by Nichodon** [comment]
Yes, I saw that font and wanted to eventually use it but I was too lazy to grab it so I just used a Google Font temporarily lol.
